frontend/src/old-pages/Configure/Queues/__tests__/SubnetMultiSelect.test.tsx

```diff
@@ -0,0 +1,180 @@
+        if (arrayEquals(path, ['app', 'wizard', 'vpc'])) {
+          return 'VPC-1'
+        } else {
+          return mockSubnets
+        }
+      })
+    })
+
+    it('should show as options only the subnets in that VPC', () => {
+      const mockedOnChange = jest.fn()
+      const {container} = render(
+        <SubnetMultiSelect
+          value={mockSelectedSubnets}
+          onChange={mockedOnChange}
+        />,
+      )
+
+      const multiSelect = wrapper(container).findMultiselect()
+      expect(multiSelect).toBeDefined()
+      multiSelect?.openDropdown()
+      expect(multiSelect?.findDropdown()?.findOptions().length).toBe(2)
+      expect(
+        multiSelect?.findDropdown()?.findOptionByValue('subnet-1'),
+      ).toBeTruthy()
+      expect(
+        multiSelect?.findDropdown()?.findOptionByValue('subnet-2'),
+      ).toBeTruthy()
+    })
+  })
+
+  describe('when only one subnet is selected', () => {
+    beforeEach(() => {
+      mockUseState.mockImplementation((path: string[]) => {
+        if (arrayEquals(path, ['app', 'wizard', 'vpc'])) {
+          return 'VPC-1'
+        } else {
+          return mockSubnets
+        }
+      })
+    })
+
+    it('should show the selected subnet only', () => {
+      const mockedOnChange = jest.fn()
+      const {container} = render(
+        <SubnetMultiSelect
+          value={mockSelectedSubnets}
+          onChange={mockedOnChange}
+        />,
+      )
+
+      const multiSelect = wrapper(container).findMultiselect()
+      expect(mockedOnChange).toHaveBeenCalledTimes(0)
+      expect(
+        multiSelect?.findToken(1)?.findLabel().getElement().textContent,
+      ).toBe('subnet-1')
+    })
+  })
+
+  describe('when no subnet is selected', () => {
+    beforeEach(() => {
+      mockUseState.mockImplementation((path: string[]) => {
+        if (arrayEquals(path, ['app', 'wizard', 'vpc'])) {
+          return 'VPC-1'
+        } else {
+          return mockSubnets
+        }
+      })
+    })
+
+    it('should not display any subnet', () => {
+      const mockedOnChange = jest.fn()
+      const {container} = render(
+        <SubnetMultiSelect value={[]} onChange={mockedOnChange} />,
+      )
+
+      const multiSelect = wrapper(container).findMultiselect()
+      expect(multiSelect?.findTokens().length).toBe(0)
+    })
+  })
+
+  describe('when the user selects multiple subnets', () => {
+    beforeEach(() => {
+      mockUseState.mockImplementation((path: string[]) => {
+        if (arrayEquals(path, ['app', 'wizard', 'vpc'])) {
+          return 'VPC-1'
+        } else {
+          return mockSubnets
+        }
+      })
+    })
+
+    it('should display the selected subnets', () => {
+      const mockedOnChange = jest.fn()
+      const {container} = render(
+        <SubnetMultiSelect value={[]} onChange={mockedOnChange} />,
+      )
+
+      const multiSelect = wrapper(container).findMultiselect()
+      multiSelect?.openDropdown()
```

Comment:
suggestion: to avoid using plenty of `?` you can take advantage of the non-null assertion operator (`!`):
```typescript
 const multiSelect = wrapper(container).findMultiselect()!
 multiSelect.openDropdown()
```